server/conditions/edge_collision_condition.go

```diff
@@ -0,0 +1,71 @@
+package conditions
+
+import (
+	"fmt"
+	"github.com/bitdecaygames/fireport/server/actions"
+	"github.com/bitdecaygames/fireport/server/pogo"
+)
+
+// EdgeCollisionCondition address the case of two players moving into adjacent spaces through the same edge (passing through each other)
+type EdgeCollisionCondition struct{}
```

Comment:
This guy is the thing that Logan mentioned checking for.  Two players that try and move through each other.  The only way this Condition works is if movement happens in unit steps.  As in, if there is a movement action that moves a player more than 1 space, this method breaks.